recipes/jsbsim/all/patches/1.1.13-0003-fix-use-t-before-auto-deduction.patch

```diff
@@ -0,0 +1,13 @@
+--- a/src/math/FGTable.cpp
```

Comment:
This patch could be sent to the upstream for sure.

Reply:
Yep, I was waiting c3i result.